server/routes/notification.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+
+const {
+    validateMakeNotification, 
+    validateMarkReadNotification,
+    validateNotification,
+} = require('../middleware/notification');
+
+const { 
+    makeNotification, 
+    markReadNotification, 
+    getAllNotifications, 
+    getUnreadNotifications 
+} = require('../controllers/notification'); 
+
+router.route("/")
+    .all(protect)
+    .get(getAllNotifications)
+    .post(validateMakeNotification, validateNotification, makeNotification);
+router.route("/mark-read")
+    .all(protect)
+    .put(validateMarkReadNotification, validateNotification, markReadNotification);
+router.route("/get-unread")
```

Comment:
Same here

Reply:
Echoing from my above comment, changing it to just `/unread`.